src/test/config.sol

```diff
@@ -122,10 +123,11 @@ contract Config {
         afterSpell.vow_dump               = 250;                            // In whole Dai units
         afterSpell.vow_sump               = 50 * THOUSAND;                  // In whole Dai units
         afterSpell.vow_bump               = 25 * THOUSAND;                  // In whole Dai units
-        afterSpell.vow_hump_min           = 55 * MILLION;                   // In whole Dai units
-        afterSpell.vow_hump_max           = 55 * MILLION;                   // In whole Dai units
-        afterSpell.split_hop              = 11_635 seconds;                 // In seconds
-        afterSpell.split_burn             = 100_00;                         // In basis points
+        afterSpell.vow_hump_min           = 60 * MILLION;                   // In whole Dai units
+        afterSpell.vow_hump_max           = 60 * MILLION;                   // In whole Dai units
+        afterSpell.split_hop              = 15_649 seconds;                 // In seconds
+        afterSpell.split_burn             = 70_00;                          // In basis points
+        afterSpell.split_farm             = "REWARDS_LSMKR_USDS";           // Farm chainlog key
```

Comment:
Shouldn't this be the actual address, same as `chief` below?

Reply:
Config contract actually doesn't have access to `addr` or other common variables available inside `DssSpellTestBase`. `chief` variable is actually a hack that is manually passed down into the `Config.setValues()` [inside DssSpellTestBase](https://github.com/makerdao/spells-mainnet/blob/625a240370b3e78fedf531de3b2dd307995d8eed/src/DssSpell.t.base.sol#L514). So looking into this, I thought providing chainlog key would serve the same function, but will be less hacky. In this context, do you see any particular problem with providing a key instead of an address here?

Reply:
Not a strong preference, just spotted the inconsistency.

Maybe "extend" the hack, injecting the address there for consistency sake, or remove the hack completely by passing the key to chief instead of injecting the address.

Reply:
Updated via b2a60339468c2fa667212c7aa1447c8f49ea99d4. Note that it also required updating starknet test due to the overlapping `setValues` methods